packages/client/utils/AtlassianClientManager.ts

```diff
@@ -14,8 +13,11 @@ class AtlassianClientManager extends AtlassianManager {
     scopes: JiraPermissionScope[] = AtlassianManager.SCOPE
   ) {
     const {submitting, onError, onCompleted, submitMutation} = mutationProps
-    const providerState = Math.random().toString(36).substring(5)
-    const redirect = makeHref('/auth/atlassian')
+    const hash = Math.random().toString(36).substring(5)
+    const providerState = btoa(
+      JSON.stringify({hash, origin: window.location.origin, service: 'atlassian'})
```

Comment:
this is the magic, the client is passing its origin & service to atlassian, which then after auth passes it to our cloud function, which then forwards it to the origin

Reply:
`appOrigin` should just be for the server going off the server's `HOST, PORT` env vars. At least I hope we haven't used it on the client anywhere :sweat_smile: 

update: we use the name `appOrigin` in SSR components like emails (https://github.com/ParabolInc/parabol/blob/b656e7d44061aabd78abadfb05e79d61de9bd181/packages/client/modules/summary/components/NewMeetingSummary.tsx#L65), but still pass in `location.origin`